scripts/translation/qaxml-pi.php

```diff
@@ -0,0 +1,71 @@
+<?php /*
++----------------------------------------------------------------------+
+| Copyright (c) 1997-2025 The PHP Group                                |
++----------------------------------------------------------------------+
+| This source file is subject to version 3.01 of the PHP license,      |
+| that is bundled with this package in the file LICENSE, and is        |
+| available through the world-wide-web at the following url:           |
+| https://www.php.net/license/3_01.txt.                                |
+| If you did not receive a copy of the PHP license and are unable to   |
+| obtain it through the world-wide-web, please send a note to          |
+| license@php.net, so we can mail you a copy immediately.              |
++----------------------------------------------------------------------+
+| Authors:     André L F S Bacci <ae php.net>                          |
++----------------------------------------------------------------------+
+
+# Description
+
+Compare processing instructions usage between two XML files.          */
+
+require_once __DIR__ . '/libqa/all.php';
+
+$argv   = new ArgvParser( $argv );
+$ignore = new OutputIgnore( $argv ); // may exit.
+$argv->complete();
+
+$list   = SyncFileList::load();
+
+foreach ( $list as $file )
+{
+    $source = $file->sourceDir . '/' . $file->file;
+    $target = $file->targetDir . '/' . $file->file;
+    $output = new OutputBuffer( "# qaxml.p" , $target , $ignore );
+
+    [ $s , $_ , $_ ] = XmlFrag::loadXmlFragmentFile( $source );
+    [ $t , $_ , $_ ] = XmlFrag::loadXmlFragmentFile( $target );
+
+    $s = XmlFrag::listNodes( $s , XML_PI_NODE );
+    $t = XmlFrag::listNodes( $t , XML_PI_NODE );
+
+    $s = extractPiData( $s );
+    $t = extractPiData( $t );
+
+    if ( implode( "\n" , $s ) == implode( "\n" , $t ) )
```

Comment:
Shouldn't this be `===`? I think it would make more sense.

Reply:
I could do the change, but as they are direct returns of identical called function (`implode`), the only case where it would compare equals with different types are false like values, basically empty strings and empty arrays. And in this case, they comparing equals is correct, as this is a fast test for "same" result on each side.

Reply:
Different ordered elements are what I was thinking. If there's no need for the elements to be in different orders then please use strict comparison.

Reply:
Changed.